ui/DesignSystem/Component/KeyHint.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+  import { isMac } from "./../../src/settings.ts";
+
+  export let hint = null;
+  export let noModifier = false;
+  export let style = null;
+
+  const modifierKey = isMac ? "⌘" : "ctrl";
```

Comment:
We don't have to do this now, but it might be nice in the future as we add more keyboard functionality: https://github.com/nfriend/ts-key-enum/blob/master/Key.enum.d.ts

Reply:
yup, next iteration.